packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -15,48 +15,59 @@ A payment network is defined by:
 
-Similar to the payment address, in this payment network, we can set a refund address that follows the same rules (it also needs to be exclusive to the request). The final balance of the request will be substracted by the inbound transfers to the refund address.
+For address based payment requests, the refund address also has to be exclusive to this payment refund.
 
 ### Reference based
 
-For this payment network, a request contains one payment address. This address doesn't have to be exclusive to the request.
-The balance is computed by reading transfers to the payment address containing a specific reference.
-The reference is a number defined by the request id and the payment address.
-There can be different ways to document the reference through the transfer. We currently define two methods that depend on the currency:
-* Through input data
-* Through a proxy smart contract
-
-Similar to the payment address, in this payment network, we can set a refund address that follows the same rules. The reference will be specific for the refunds.
-
-#### Input data:
-
-In certain cases, when transferring a currency amount, the user has the choice to add additional data to the transaction. For example, Ethereum allows the user to add miscellaneous data named *input data* when performing a simple ether transfer.
-In this case, the payment reference is documented here.
-
-#### Proxy smart contract:
-
-In this case, the reference is documented through a proxy smart contract.
-This is a smart contract that forwards a currency transfer and stores a reference.
-The currency must be backed by a blockchain with smart contract capabilities.
+For these payment networks, a request contains one payment address. This address doesn't have to be exclusive to the request.
+The balance is computed by reading transfers to the payment address containing a specific payment reference, defined by the request ID and payment address.
+
+For ETH, we can tag and detect the payment reference directly in the transaction, using the `input data` field of the transaction.
+
+When we cannot use `input data` or equivalent, typically for ERC20, we use a *proxy smart contract* to document the payment reference.
+The smart contract forwards a currency transfer and stores a reference.
+
+If you need the proxy smart contract addresses, we list the most relevant ones below, together with Github links to find more details and up-to-date information.
+
+[Proxy smart contracts for ERC20](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/smart-contracts/artifacts/ERC20Proxy/artifacts.json):
+```json
+"mainnet": {
+	"address": "0x5f821c20947ff9be22e823edc5b3c709b33121b3",
+},
+"rinkeby": {
+	"address": "0x162edb802fae75b9ee4288345735008ba51a4ec9",
+}
+```
+[Proxy smart contracts for ETH](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/smart-contracts/artifacts/EthereumProxy/artifacts.json):
```

Comment:
this could use some more information, as it is explained right above that ETH can use the input data. @vrolland , did we do this to enable multisig? I don't remember...

Reply:
Yes, I wondered the same. I guess there can be a use... Maybe just
```suggestion
[Proxy smart contracts for ETH when input data cannot be used](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/smart-contracts/artifacts/EthereumProxy/artifacts.json):
```

Reply:
Yep, it is to pay from a smart-contract. with only the input data it is very tricky to detect the payment as it is in an internal transaction.